src/styles/blocks/header.scss

```diff
@@ -0,0 +1,52 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: $clear_white-color;
+  box-shadow: 0 2px 4px rgba(0,0,0,0.05);
+  margin: 0 auto;
+
+  &__image {
+    height: 40px;
+    width: 40px;
+  }
+}
```

Comment:
Same here. Better to drop them into the other file.

Reply:
agree, 1 block — 1 file

Reply:
fixed